runelite-client/src/main/java/net/runelite/client/plugins/microbot/fishing/barbarian/BarbarianFishingScript.java

```diff
@@ -8,28 +8,31 @@
 import net.runelite.client.plugins.microbot.util.inventory.DropOrder;
 import net.runelite.client.plugins.microbot.util.inventory.Rs2Inventory;
 import net.runelite.client.plugins.microbot.util.npc.Rs2Npc;
-import net.runelite.client.plugins.microbot.util.player.Rs2Player;
```

Comment:
You removed a reference to Rs2Player but it's still being used inside the class.

Reply:
Oops, my bad, its fixex now